### ISSUE TITLE
📆 "Upcoming cards": Display list of each card

### ISSUE THREAD

**Issue by MartinBK25** [nextcloud/deck#2770]
Hey,

as a user i want to see the list of each card in the "Upcoming cards" view.

My current workaround is to add and remove tags with the same name as the list to each card because they are always visible in the "Upcoming cards" view.

Thanks 

<details>
<summary>Original request</summary>
It would be great to see current status on "Upcoming cards". Right now you have to tag it manually als "todo","done" or etc. .
The column could be use to tag cards automatically,
</details>

**Comment by stefan-niedermann** [comment]
If i understand you correctly, you want to **add a tag to a card if you move the card to a column - and remove the tag if you move the card to another column**?

This doesn't make sense at all for me - why duplicating the information?
But you will for sure have your reasons / your workflow.

This sounds like the kind of automation which should be possible using Nextcloud Flow - I propose to add this use case the [Flow integration issue](https://github.com/nextcloud/deck/issues?q=is%3Aissue+is%3Aopen+flow) because

> Anyone is free to come up with additional use cases that might be useful for automating tasks around deck

and i honestly think this is rather an edge case which should not be hard coded into the Deck core.

**Comment by MartinBK25** [comment]
`If i understand you correctly, you want to add a tag to a card if you move the card to a column - and remove the tag if you move the card to another column?`
that's right

It's to see the status at "Upcoming cards". To see the status you have to add manually tags.
Maybe this tag should only visible at "Upcoming cards".

**Comment by stefan-niedermann** [comment]
Isn't the actual feature request simply "**I want to see the list of each card in the Upcoming cards view**" then? (Instead of any add- and remove-magic with tags)

**Comment by MartinBK25** [comment]
yes, I want so see the current state/column of a card in Upcomming cards view.

you understand me :-D

**Comment by jakobroehrl** [comment]
@MartinBK25 
Do you mean this? https://github.com/nextcloud/deck/issues/2235

**Comment by stefan-niedermann** [comment]
Ah yes, it's a duplicate which i missed! Let's track the progress there.

**Comment by stefan-niedermann** [comment]
Feel free to reopen the issue, but i strongly vote against the same visual representation as tags. Lists (= Stacks) are **not** tags and should therefore not look like the same. Otherwise users can not distinguish tags from the lists.

**Comment by MartinBK25** [comment]
It doesn't have to be a tag. I thought tag would be a good idea because it already exists and it is human readable.
I would like to see it as a text and not as a symbol or icon.
